settings.php

```diff
@@ -26,12 +26,18 @@
 defined('MOODLE_INTERNAL') || die();
 
 if ($hassiteconfig) {
-    $ADMIN->add('reports', new admin_category('tool_cloudwatch_reports', 'Cloudwatch'));
+    $ADMIN->add('reports', new admin_category('tool_cloudmetrics_reports', get_string('pluginname', 'tool_cloudmetrics')));
```

Comment:
the settings should be nested under Admin > Plugins > Admin tool > ...

The reports themselves might later be under the reports tab though